tests/unit/search/test_search_queries.py

```diff
@@ -0,0 +1,38 @@
+# -*- coding: utf-8 -*-
+#
+# This file is part of INSPIRE.
+# Copyright (C) 2016 CERN.
+#
+# INSPIRE is free software; you can redistribute it
+# and/or modify it under the terms of the GNU General Public License as
+# published by the Free Software Foundation; either version 2 of the
+# License, or (at your option) any later version.
+#
+# INSPIRE is distributed in the hope that it will be
+# useful, but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the GNU
+# General Public License for more details.
+#
+# You should have received a copy of the GNU General Public License
+# along with INSPIRE; if not, write to the
+# Free Software Foundation, Inc., 59 Temple Place, Suite 330, Boston,
+# MA 02111-1307, USA.
+#
+# In applying this license, CERN does not
+# waive the privileges and immunities granted to it by virtue of its status
+# as an Intergovernmental Organization or submit itself to any jurisdiction.
+
+"""Test elasticsearch query production."""
+
+from inspirehep.modules.search.query import InspireQuery
+
+
+def test_fc_keyword():
+    """
+    Test queries using field_code keyword (fc) .
+    """
+    query_using_spires_syntax = InspireQuery('find fc a')
+    query_using_invenio_syntax = InspireQuery('fc: a')
+
+    assert query_using_spires_syntax.body == {'query': {'multi_match': {'query': 'a', 'fields': ['field_code']}}}
+    assert query_using_invenio_syntax.body == {'query': {'multi_match': {'query': 'a', 'fields': ['field_code']}}}
```

Comment:
This approach is kind-of-superseded by the one in #1016, which will be merged soon™. Only the second test will survive as `test_field_code_colon` and with a structure similar to the other tests in `tests/unit/search/test_search_query.py`.